README.md

```diff
@@ -1,2 +1,166 @@
 # kbcli
-A Command Line Interface for KubeBlocks
+
+kbcli is a command line interface (CLI) tool for [KubeBlocks](https://github.com/apecloud/kubeblocks). kbcli has the following features:
+
+- Manage KubeBlocks, including installation, uninstallation, viewing status, and upgrading, etc.
+- Manage clusters, including creating, deleting, configuration changes, backups and restores, etc.
+- Support playgrounds to quickly experience KubeBlocks locally or on the cloud.
+
+## Install kbcli
+
+### Install kbcli on Linux
+
+#### Install with curl
+
+Install the latest linux kbcli to `/usr/local/bin`
+
+```bash
+curl -fsSL https://kubeblocks.io/installer/install_cli.sh | bash
+```
+
+#### Install using native package management
+
+**Debian-based distributions**
+
+TODO
```

Comment:
can add the installation of using apt in the [readme](https://github.com/apecloud/kbcli-apt).